eip-1175.md

```diff
@@ -0,0 +1,529 @@
+---
+eip: 1175
+title: Wallet & shop standard for all tokens (erc20)
+author: Jet Lim (@Nitro888)
+status: Draft
+type: Standards Track
+category: ERC
+created: 2018-06-21
+requires: 20
+---
+
+# All tokens go to heaven
+## Simple Summary
+Make wallets and shops created from certified contracts make erc20 tokens easy to use for commerce.
+
+![wallet](https://user-images.githubusercontent.com/11692220/41762799-ee17c480-7636-11e8-9930-681be2c59b56.png)
+
+## Abstract
+The mutual trust between the wallet and the shop created by the authenticated contract allows you to pay for and purchase items at a simple process.
+
+## Motivation
+We want to make it easy to trade items by creating a promised interface of wallets and stores that can build mutual trust. Because it is a mutual trust wallet and store, the transaction is made by calling `pay(address _shop, uint256 _item)`once without using `approve(address _spender, uint256 _value)` and `transferFrom(address _from, address _to, uint256 _value)`.
```

Comment:
This needs expanding - it's unclear how this would work. Can you add a section to the start of 'specification' that describes what the intended workflow is here?